src/test/java/core/basesyntax/strategy/SupplyStrategyTest.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class SupplyStrategyTest {
+    private final Strategy strategy = new SupplyStrategy();
+
+    @Test
+    void calculateFruitQuantity() {
+        Integer num = 100;
+        Integer num2 = 20;
```

Comment:
Please rename variables to add some context

Reply:
fixed